src/Components/Header/Header.jsx

```diff
@@ -151,49 +156,71 @@ function Header({ program, loadProgram, history, admin, curRoute, routes }) {
             </div>
           </Tooltip>
         </AppName>
-        <p>
-          {" • "}
```

Comment:
No more dot. Intentional?

Reply:
Oh, I thought the dot might be getting replaced with a coloured dot with the customization stuff. Wasn't sure though.

Reply:
Oh interesting, perhaps. I thought the coloured dot was just for form customization, not the navbar too. @GodGreg or @brittanylau, can you comment on the expected look here?

Reply:
Designs have a dot. Brittany had mentioned that we can exclude it.